distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Extensions.enso

```diff
@@ -1894,6 +1895,46 @@ Text.parse_time_of_day self format:Date_Time_Formatter=Date_Time_Formatter.iso_t
 Text.parse_time_zone : Time_Zone ! Time_Error
 Text.parse_time_zone self = Time_Zone.parse self
 
+## GROUP Conversions
+   ICON convert
+   Convert this `Text` to a `Decimal`.
+
+   Arguments:
+   - scale: the optional Decimal scale to use. See `Decimal.set_scale` for more
```

Comment:
Do we have a way to link directly to other parts of our documentation?

Reply:
Not really at the moment.

We should use the Markdown syntax and could pick up in the website version and then think about adding to the doc panel as well.